packages/overlay/test/restore-focus.test.js

```diff
@@ -53,52 +48,16 @@ describe('restore focus', () => {
     document.body.focus();
   });
 
-  it('should not restore focus on close by default (restore-focus-on-close=false)', async () => {
-    overlay.restoreFocusOnClose = false;
-    focusInput.focus();
-    await open(overlay);
-    // Emulate focus leaving the input
-    focusInput.blur();
-    document.body.focus();
-    await close(overlay);
-    expect(overlay._getActiveElement()).to.not.equal(focusInput);
-  });
-
-  it('should restore focus on close (restore-focus-on-close=true)', async () => {
-    overlay.restoreFocusOnClose = true;
-    focusInput.focus();
-    await open(overlay);
-    // Emulate focus leaving the input
-    focusInput.blur();
-    document.body.focus();
-    await close(overlay);
-    expect(overlay._getActiveElement()).to.equal(focusInput);
-  });
-
-  it('should restore focus on close in Shadow DOM (restore-focus-on-close=true)', async () => {
-    overlay.restoreFocusOnClose = true;
-    focusable.focus();
-    await open(overlay);
-    await close(overlay);
-    expect(overlay._getActiveElement()).to.equal(focusable);
-  });
-
-  it('should not restore focus on close if focus was changed', async () => {
-    overlay.restoreFocusOnClose = true;
-    focusInput.focus();
-    await open(overlay);
-    focusable.focus();
-    await close(overlay);
-    expect(overlay._getActiveElement()).to.equal(focusable);
-  });
+  describe('basic', () => {
```

Comment:
**note:** I'm planning to add another test suite "slotted focus node" for verifying that focus-ring is restored on the host component rather than the focus node itself in that case.